tests/framework/e2e/etcdctl.go

```diff
@@ -292,13 +292,13 @@ func (ctl *EtcdctlV3) Status() ([]*clientv3.StatusResponse, error) {
 		Endpoint string
 		Status   *clientv3.StatusResponse
 	}
-	err := ctl.spawnJsonCmd(&epStatus, "endpoint", "status", "--endpoints", strings.Join(ctl.endpoints, ","))
+	err := ctl.spawnJsonCmd(&epStatus, "endpoint", "status")
```

Comment:
good!!!

It's unnecessary, because [(*EtcdctlV3) flags()](https://github.com/etcd-io/etcd/blob/216f0ae7e94d33bd97e51da119a674fdfc578050/tests/framework/e2e/etcdctl.go#L274) adds `"--endpoints"`